zorm/src/main/kotlin/com/zenmo/orm/user/User.kt

```diff
@@ -1,8 +1,11 @@
 package com.zenmo.orm.user
 
+import com.zenmo.zummon.companysurvey.Project
+import java.util.UUID
+
 data class User(
     // Keycloak id
-    val id: String,
-    val projects: List<String>,
+    val id: UUID = UUID.randomUUID(),
```

Comment:
The ID is always from Keycloak so it shouldn't be generated.

Reply:
Ok.